src/Forms/NumericField.php

```diff
@@ -154,6 +154,9 @@ public function setValue($value, $data = null)
      */
     protected function cast($value)
     {
+        if (strlen($value) === 0) {
+            return false;
```

Comment:
It's set to false, so that in the Value function it doesn't end up being pushed into a formatter.

I can update the Value function to check is null and false if you'd prefer.